xml/ay_bigfile.xml

```diff
@@ -17165,44 +17161,65 @@ mv /var/run/zypp.sav /var/run/zypp.pid
   <title>Advanced Linuxrc Options</title>
   <info/>
   <para>
-   Linuxrc is a program used for setting up the kernel for installation
-   purposes. It allows the user to load modules, start an installed system,
-   a rescue system or an installation via &yast;.
+   <command>Linuxrc</command> is a small program that runs after the kernel loads, but before &ay;
+   or other stages. It prepares the system for installation. It allows the user
+   to load modules, start an installed system or a rescue system, and to guide
+   the operation of &yast;.
   </para>
+  <note>
+   <title>Linuxrc Settings Are Not The Same As &ay; Settings</title>
+   <para>
+    Some <command>linuxrc</command> settings have the same names as settings
+    used by &ay; in the <filename>autoyast.xml</filename> file. Beware that this
+    does not mean that they take the same parameters or function in the same
+    way. For example, &ay; takes a parameter <command>self_update</command>
+    and if this is set, another called <command>self_update_url</command>.
+    Linuxrc also has a <command>self_update</command> parameter, but this takes
+    either <literal>0</literal> or a URL. Attempting to pass &ay; parameters to
```

Comment:
I would make it a clear instruction: "Do not pass &ay; parameters to ```<command>linuxrc</command>``` as it will not give the desired results."